build.gradle.kts

```diff
@@ -118,14 +119,20 @@ val downloadPlantUml = tasks.register<NpmTask>("downloadPlantUml") {
 
 tasks.register<NodeTask>("antora") {
     setDependsOn(listOf(downloadAntoraCli, downloadAntoraSiteGeneratorLunr, downloadPlantUml))
-    setScript(file("${project.buildDir}/node_modules/@antora/cli/bin/antora"))
+    script = file("${project.buildDir}/node_modules/@antora/cli/bin/antora")
     setArgs(listOf("--generator", "antora-site-generator-lunr", "site.yml", "--stacktrace"))
     setWorkingDir(project.projectDir)
 }
 
 tasks.register<NodeTask>("antoraLocal") {
     setDependsOn(listOf(downloadAntoraCli, downloadAntoraSiteGeneratorLunr, downloadPlantUml))
-    setScript(file("${project.buildDir}/node_modules/@antora/cli/bin/antora"))
+    script = file("${project.buildDir}/node_modules/@antora/cli/bin/antora")
     setArgs(listOf("--generator", "antora-site-generator-lunr", "site-local.yml", "--stacktrace"))
     setWorkingDir(project.projectDir)
 }
+
+tasks.findByName("publishToMavenLocal").also {
```

Comment:
It seems like this does not only affect the `publishToMavenLocal` but also other tasks, such as `publishMavenReleasePublicationToMavenRepository`, which we use in the release workflow. When removing these lines I get an error `Could not read PGP secret key` (using and invalid key/password), indicating that the signing process was started. When I don't remove the code I immediately get `401 - Unauthorized`, indicating that the signing was skipped and instead the actual publish did not succeed (because of invalid credentials).
Therefore, I guess we would run into a problem in the next release with these changes.

Reply:
Good catch! Apparently `Tasks.also` behaves differently than I expected. I rewrote it now to use `dependsOn` instead and verified that `./gradlew publishMavenReleasePublicationToMavenRepository` tries to sign while `./gradlew publishToMavenLocal` does not.